roles/repositories/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+repositories_epel: true
+repositories_puppet: true
+repositories_foreman: true
```

Comment:
I'm not sure anything but EPEL should be true by default - and not even sure about what.  I'm thinking boxes like katello_client, or things that might not need puppet and foreman repos.

Reply:
That's an interesting one. I had the foreman server repositories in mind with optional extensions for katello. Now a name `server_repositories` is also ambiguous but I'd like to avoid a role that can serve both clients and servers. If we look at the uses of repositories there's little overlap so one role to rule them all just adds confusion IMHO.